pkg/rtc/mediatrackreceiver.go

```diff
@@ -568,9 +568,15 @@ func (t *MediaTrackReceiver) TrackInfo(generateLayer bool) *livekit.TrackInfo {
 					ci.Layers = append(ci.Layers, proto.Clone(layer).(*livekit.VideoLayer))
 
 					// if origin layer has ssrc, don't override it
-					if layerIdx < len(originLayers) && originLayers[layerIdx].Ssrc != 0 {
-						ci.Layers[layerIdx].Ssrc = originLayers[layerIdx].Ssrc
-					} else if int(layer.Quality) < len(receiver.layerSSRCs) {
+					ssrcFound := false
```

Comment:
@cnderrauber I was just about to open a PR to do the same thing :-). Will review this and abort my attempt.